interfaces/Portal/src/app/app.component.ts

```diff
@@ -32,14 +33,23 @@ export class AppComponent implements OnInit, OnDestroy {
     }
   }
 
-  public ngOnInit(): void {
+  public async ngOnInit() {
```

Comment:
```suggestion
  public ngOnInit() {
```

I don't see an `await` so I don't think this is needed